tests/test-data/pkg/expected-warnings-darwin.txt

```diff
@@ -92,3 +84,8 @@
   The file must be distributed with executable as %2.
   %1: node_modules/opn/xdg-open
   %2: path-to-executable/xdg-open
+prebuild-install warn install No prebuilt binaries found (target=v16.16.0 runtime=node arch=x64 libc= platform=darwin)
```

Comment:
I suspect we need to solve this warning instead of adding it to the expected list. I don't understand enough yet about how `prebuild-install` works

Reply:
apparently not, the warning is harmless